### PR TITLE
TEST DIFF!!! Do not merge

### PR DESCRIPTION
**Pull request by ann0see**
Diff from known last working state

### CODE REVIEW

File: wiki/en/Running-a-Server.md [hunk 1]
```diff
@@ -29,39 +29,22 @@ If you decide you cannot use any of the Servers listed by the built in Directori
...
-
-If you plan to be playing regularly with the same people, **you are strongly advised** to first make sure that each member of the group is set up to use Jamulus properly. Do this by finding a Public Server with a reasonable ping time for all of you (20 ms or less perhaps), all connect to that and work to fix any individual issues (verifying that they can [follow Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) in particular). Use the solo technique above to prevent being interrupted if needed.
-
-Once any issues with musicians have been solved in this way, you can then investigate hosting your own Server either at home or on a cloud host such as Amazon, which may result in better latency than Servers run at home.
-
-### Bandwidth – do you have enough?
-
-Unless you plan on hosting more than about 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up), then you are unlikely to run out of bandwidth. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
 
 ### In general
```
Comment (pljones) [inline wiki/en/Running-a-Server.md:33]: Probably wants to be L2; don't like the title.

File: wiki/en/Running-a-Server.md [hunk 2]
```diff
@@ -29,39 +29,22 @@ If you decide you cannot use any of the Servers listed by the built in Directori
...
-
-- Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the Client and Server. There are plans to expand IPv6 support.
+* Consider using a cloud host to get better ping times if you're having problems
+* Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
+* Running a Server may require you to adjust any firewalls running on or outside of your machine or cloud host.
+* Running an Unregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+* Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the Client and Server. There are plans to expand IPv6 support.
 
 # Server Types
```
Comment (pljones) [inline wiki/en/Running-a-Server.md:41]: Definitely wants to be L2.  There should be no L1 headings throughout.

File: wiki/en/Running-a-Server.md [hunk 3]
```diff
@@ -83,69 +66,34 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
 
 To run a Custom Directory [read this guide](Custom-Directories)
 
+### Speed and latency
```
Comment (pljones) [inline wiki/en/Running-a-Server.md:69]: Doesn't belong under "Server Types".

File: wiki/en/Running-a-Server.md [hunk 4]
```diff
@@ -83,69 +66,34 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+Please see the [Server Installation page](Server-Install).
 
 ### Maintenance
 
 #### Upgrading
 
-To upgrade your Server to a newer version, simply download a new .deb and [re-install as step 3](#installation).
+To upgrade your Server to a newer version, simply download a new .deb and [re-install as step 3 in the install guide](Server-Install#installation).
 
 #### Viewing The Logs
```
Comment (pljones) [inline wiki/en/Running-a-Server.md:94]: Not really "Maintenance".

File: wiki/en/Running-a-Server.md [hunk 5]
```diff
@@ -83,69 +66,34 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
 
-Jamulus will log to the system log file if you left the `StandardOutput=journal` setting in the unit file. 
+Jamulus will log to the system log file if you left the `StandardOutput=journal` setting in the unit file.
 
 To view the log, use `journalctl` (to exit press Ctrl-C). For example, to read the system log file, filtered for the Jamulus service:
```
Comment (pljones) [inline wiki/en/Running-a-Server.md:99]: Line 102 (new) definitely isn't Maintenance.

File: wiki/en/Running-a-Server.md [hunk 6]
```diff
@@ -216,15 +164,6 @@ function loadserverstat ( $statfilename )
...
-Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
-
-* **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
-
-* **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
-
-* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return. _(To run a headless Server [read this guide](#installing-and-running-a-server))_
 
 
 ## Server Setup
```
Comment (pljones) [inline wiki/en/Running-a-Server.md:169]: Each of the following Server Setup options should show _both_ how to do it from the UI and how to do it from the command line.

Comment (pljones) [reply wiki/en/Running-a-Server.md:169]: Line 169 (new) is refering to the _tab name_ in the UI, of course, as is line 191 (new), so these headings need addressing.

File: wiki/en/Server-Install.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
+1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
+
+**Note**: You can upgrade your server by repeating the steps mentioned above.
+
+### Configuration
```
Comment (pljones) [inline wiki/en/Server-Install.md:30]: Not relevant to this page.

File: wiki/en/Server-Install.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
+
+`sudo systemctl stop jamulus-headless`
+
+### Running an Unregistered Server
```
Comment (pljones) [inline wiki/en/Server-Install.md:36]: Not relevant to this page.

File: wiki/en/Server-Install.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+## Dynamic DNS and why you will probably need it
+
+Home internet connections ususally change their IP address after a short period (hours, days or weeks). To allow people to connect to you, you probably want set up a dynamic DNS address for getting a static (sub-)domain people can connect to. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
+
+**Note:** Your home router may change the IP address of the machine you're running your Server on. You may need to give that machine a static IP in the router's DHCP configuration.
+
+#### Running in Registered mode
```
Comment (pljones) [inline wiki/en/Server-Install.md:56]: From here down is not relevant to this page.

File: wiki/en/Running-a-Server.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Running-a-Server.md:33]: Changed in #725 to General notes

File: wiki/en/Running-a-Server.md (hunk 2 above)
Comment (ann0see) [reply wiki/en/Running-a-Server.md:41]: Yes. Fixed that in the other PR.

File: wiki/en/Running-a-Server.md (hunk 4 above)
Comment (ann0see) [reply wiki/en/Running-a-Server.md:94]: Please comment this in #725

File: wiki/en/Running-a-Server.md (hunk 6 above)
Comment (ann0see) [reply wiki/en/Running-a-Server.md:169]: @gilgongo I think that's worth another PR/issue.

**Comment by ann0see** [comment]
Closing in favor of #725 and the hopefully fixed GitHub hiccups.

**Pull request closed without merging**